clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationClient.ts

```diff
@@ -100,6 +102,7 @@ export class BrowserAuthorizationClient extends BrowserAuthorizationBase<Browser
       post_logout_redirect_uri: basicSettings.postSignoutRedirectUri, // eslint-disable-line @typescript-eslint/naming-convention
       response_type: basicSettings.responseType, // eslint-disable-line @typescript-eslint/naming-convention
       automaticSilentRenew: true,
+      silent_redirect_uri: basicSettings.silentRedirectUri ?? basicSettings.redirectUri, // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
@evelynpreslar-bentley thoughts? 

```suggestion
      silent_redirect_uri: basicSettings.silentRedirectUri, // eslint-disable-line @typescript-eslint/naming-convention
```

Reply:
This is a fine change.
Simply enabling this option might be useful outside of setting it via advanced settings at some point.